Description

Discovered in some q test run:

 TestCliDriver.testCliDriver_insert_values_orig_table:123-&amp;gt;runTest:199 Unexpected exception java.util.ConcurrentModificationException
	at java.util.HashMap$HashIterator.nextEntry(HashMap.java:926)
	at java.util.HashMap$EntryIterator.next(HashMap.java:966)
	at java.util.HashMap$EntryIterator.next(HashMap.java:964)
	at org.apache.hadoop.hive.ql.metadata.Hive.dumpAndClearMetaCallTiming(Hive.java:3412)
	at org.apache.hadoop.hive.ql.Driver.dumpMetaCallTimingWithoutEx(Driver.java:574)
	at org.apache.hadoop.hive.ql.Driver.execute(Driver.java:1722)
	at org.apache.hadoop.hive.ql.Driver.runInternal(Driver.java:1342)
	at org.apache.hadoop.hive.ql.Driver.run(Driver.java:1113)
	at org.apache.hadoop.hive.ql.Driver.run(Driver.java:1101)




